dhall-docs/src/Dhall/Docs.hs

```diff
@@ -81,6 +75,36 @@ parseOptions =
        <> Options.Applicative.metavar "OUTPUT"
        <> Options.Applicative.help "Directory where your docs will be generated"
        <> Options.Applicative.value "docs" )
+    <*> parsePackageNameResolver
+
+  where
+    parseStrategy :: Parser GenerationStrategy
+    parseStrategy =
+        Options.Applicative.flag
+            MultiPage
+            SinglePage
+            (   Options.Applicative.long "single-page"
+            <>  Options.Applicative.help
+                    (  "Generate a single page HTML documentation. By default, "
+                    <> "the tool will generate a multi-page documentation"
+                    )
+            )
+
+    parsePackageNameResolver :: Parser (Path Abs Dir -> String)
+    parsePackageNameResolver = fmap f (Options.Applicative.optional p)
+      where
+        f  Nothing = init . Path.fromRelDir . Path.dirname
+        f (Just packageName) = const packageName
+
+        p = Options.Applicative.strOption
+                (   Options.Applicative.long "package-name"
+                <>  Options.Applicative.metavar "PACKAGE-NAME"
+                <>  Options.Applicative.help
+                            (  "Override for the package name seen on HTML "
+                            <> "navbars. By default, it will extract it from "
+                            <> "the dirname of the --input flag"
```

Comment:
Not sure whether that's better…

```suggestion
                            <> "navbars. By default, it will extract it the "
                            <> "input directory name"
```

Reply:
It is shorter and emits the same thought, so I'll keep your suggestion!